### PR TITLE
Feature/182-add-fingerprint-method

### LINKED ISSUE #182: Add method for getting the TLS certificate fingerprint
Tests usually need the fingerprint to configure DB connections etc. Currently you need to parse the JDBC URL which causes code duplication.

### PR DESCRIPTION
**Pull request by kaklakariada**
Closes #182

### CODE REVIEW

File: src/main/java/com/exasol/containers/ExasolContainer.java [hunk 1]
```diff
@@ -892,4 +890,15 @@ public String getHostIp() {
     public Optional<X509Certificate> getTlsCertificate() {
         return this.certificateProvider.getCertificate();
     }
+
+    /**
+     * Get the SHA256 fingerprint of the self-signed TLS certificate used by the database in the container for database
```
Comment (redcatbear) [inline src/main/java/com/exasol/containers/ExasolContainer.java:895]:
```suggestion
     * Get the SHA256 fingerprint of the TLS certificate used by the database in the container for database
```
This isn't necessarily always self-signed. You can replace it with a proper one.

File: src/main/java/com/exasol/containers/ExasolContainer.java [hunk 2]
```diff
@@ -892,4 +890,15 @@ public String getHostIp() {
...
+     * connections and the RPC interface.
+     *
+     * @return the SHA256 fingerprint of the TLS certificate or an empty {@link Optional} when the certificate file does
```
Comment (redcatbear) [inline src/main/java/com/exasol/containers/ExasolContainer.java:898]:
```suggestion
     * @return SHA256 fingerprint of the TLS certificate or an empty {@link Optional} when the certificate file does
```

File: src/test/java/com/exasol/containers/ExasolContainerVersionSpecificTest.java [hunk 3]
```diff
@@ -22,11 +20,10 @@
 
     @Test
     void testContainer62x() {
-
-        ExasolContainer<? extends ExasolContainer<?>> containerV62 = new ExasolContainer<>("6.2.7-d1", false);
-        var exception = assertThrows(ContainerLaunchException.class, containerV62::start);
-        assertThat(exception.getMessage(), containsString("E-ETC-13"));
-
+        try (ExasolContainer<? extends ExasolContainer<?>> containerV62 = new ExasolContainer<>("6.2.17-d1", false)) {
```
Comment (redcatbear) [inline src/test/java/com/exasol/containers/ExasolContainerVersionSpecificTest.java:23]: Funny, I just fixed the same thing in my branch. :-)

**Review by redcatbear — CHANGES_REQUESTED** [review]

File: src/test/java/com/exasol/containers/ExasolContainerVersionSpecificTest.java (hunk 3 above)
Comment (kaklakariada) [reply src/test/java/com/exasol/containers/ExasolContainerVersionSpecificTest.java:23]: 😆

**Review by redcatbear — APPROVED** [review]

**Pull request merged**